src/platform/plugins/shared/inspector/public/types.ts

```diff
@@ -62,6 +62,7 @@ export interface InspectorOptions {
   title?: string;
   options?: unknown;
   flyoutType?: EuiFlyoutProps['type'];
+  uuid?: string;
```

Comment:
nit: The naming of this could be a little better. It's described nicely in the `openLazyFlyout` types, but here it seems a little out of place. Especially since it's irrelevant when the inspector is used outside of a Dashboard context. I wonder if it might be possible to name this option differently?

Reply:
do you think we should follow the `focusedPanelId`? This is essentially what adding this does 🤔  Open to other ideas too!

Reply:
I think that works yeah. It's a tiny bit of a code smell to me that this highly dashboard-specific prop is being leaked into the generic inspector opening function., but I don't see a way around it.